tests/test_measurements.py

```diff
@@ -349,7 +349,7 @@ def circuit():
                     ),
                 ],
                 [0.3, 1.0],
-                0.9319728930156066,
+                0.9319728930156066 if device_name != "lightning.tensor" else 1.0,
```

Comment:
:thinking:

Reply:
It was a bug... The result should `1.0` for `lightning.tensor` since the Hermitian is set as identity.